extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js

```diff
@@ -125,11 +125,10 @@ fairDataPointPostDialog.prototype._editBaseUri = function(src) {
 			self._distributionDiv.html('');
 			self._pushtoResourceDiv.html('');
 			self.uri = self._baseUriSpan.text();
-			getFairCatalogs(self._baseUriSpan.text(), self);
-			return;
 		}
 		MenuSystem.dismissAll();
 		self._replaceBaseUri(newBaseUri, false);
+		getFairCatalogs(self._baseUriSpan.text(), self);
```

Comment:
Could you explain a bit more about this fix? When was the if clause above called? As far as I can see, the baseuri length > 7 conditional is true even if you add a new url?

Reply:
the clause is called when the user presses the apply button within the push to fdp dialog, the user first presses the edit button to change the baseuri. The `self._replaceBaseUri` call takes care off setting the new `self.fairDataPointPost.baseUri`